picard/acoustid/json_helpers.py

```diff
@@ -34,16 +34,48 @@ def _make_releases_node(recording):
                 release_mb['title'] = release['title']
             else:
                 release_mb['title'] = release_group['title']
+
             if 'country' in release:
                 release_mb['country'] = release['country']
+
+            if 'date' in release:
+                release_mb['date'] = release['date']
+
+            if 'medium_count' in release:
+                release_mb['medium_count'] = release['medium_count']
```

Comment:
The idea here is to bring the data returned by the AcoustID service into the same format as the JSON result from the MB web service. But there is actually no `medium_count` attribute on the MB web service. If this is needed it can be taken from the length of the `media` list.

I would vote for not including this attribute here. But if we do it should be renamed `medium-count`.

Reply:
Done.  renamed to media-count, and used the length of the list as suggested.